Add `Wiki maintainers` page (#25)
osuTitanic/wiki: wiki/People/Wiki_Maintainers/en.md

> ## Wiki Maintainers

Reviewer: ```suggestion
## Wiki maintainers
```

> Wiki Maintainers are a group of players who are in charge of maintaing the wiki and merging changes into the offical Titanic! wiki.

Reviewer: ```suggestion
Wiki Maintainers are a group of users who are in charge of maintaing the wiki and merging changes into the official Titanic! wiki.
```

> ## Roles and Responsibilities

Reviewer: ```suggestion
## Roles and responsibilities
```

> Wiki Maintainers are responsible for merging new pull requests and making sure there are no errors.

Reviewer: ```suggestion
Wiki maintainers are responsible for reviewing and merge pull requests to make sure there are no errors.
```

## Current Members

User | Added Date 
---|---

> [BlueChinchompa](https://osu.titanic.sh/u/40)   | Since The Beginning 

Reviewer: Levi should know about this info, it would be best to wait for review

Author: I agree I tried searching around in the `#Wiki` channel in discord and any mention of maintainer/invite but I didn't find anything.

Reviewer: oh wait, you may find it on your email address which is connected to your github, perhaps try to find it on gmail or something

Reviewer 2: It would be on your gmail, yeah

> [iSlodinx](https://osu.titanic.sh/u/869) | Octobor 19th, 2024

Reviewer: ```suggestion
[iSlodinx](https://osu.titanic.sh/u/869) | October 19th, 2024
```

> [Meru](https://osu.titanic.sh/u/41) | Octobor 19th, 2024

Reviewer: ```suggestion
[Meru](https://osu.titanic.sh/u/41) | October 19th, 2024
```
